src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -39,22 +39,17 @@ def handle_long_running_operation_exception(ex):
         ex,
         fault_type='failed-long-running-operation',
         summary='Unexpected client exception in {}.'.format(LongRunningOperation.__name__))
+
     message = getattr(ex, 'message', ex)
     error_message = 'Deployment failed.'
 
     try:
-        correlation_id = json.loads(ex.response.content.decode())['properties']['correlationId']
+        correlation_id = ex.response.headers['x-ms-correlation-request-id']
```

Comment:
This should probably be a `.get` so in the event the header doesn't have this, the CLI won't crash and burn.

Reply:
Thats the point of the try statement?
I can do a `.get` instead and remove the try statement but it seems consistent with the rest of the method.